MediaRecorder.html

```diff
@@ -215,52 +221,25 @@
     </dl>
 </section>
 
-<section id="properties">
-        <h3>MediaRecorder  Properties</h3>
-        <p>
-          [TO DO: Provide API surface to set and read these. When that is done,
-          this section will be removed.]
-          </p>
-     <table class="simple">
-        <thead>
-          <tr>
-            <th>Property Name</th>
-            <th>Values</th>
-            <th>Notes</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr id="def-property-MimeType">
-            <td>MimeType</td>
-            <td><code>DOMString</code></td>
-            <td> The MIME type that was be selected as the container for recording. This entry includes all the parameters to the base MimeType.
-                The UA should be able to play back any of the MIME types it supports for recording. For example, it should be able to display a video recording in the HTML &lt;video&gt; tag.
-                The default value for this property is platform-specific.</td>
-          </tr>
-                   <tr id="def-property-BitRate">
-            <td>BitRate</td>
-            <td><code>long</code></td>
-            <td> The bitrate type that was selected as for recording. The default
-                value for this property is platform-specific.</td>
-          </tr>
-            <tr id="def-property-IgnoreMutedMedia">
-            <td>ignoreMutedMedia</td>
-            <td><code>boolean</code></td>
-            <td> If this property is set to 'true', the MediaRecorder should stop
-                recording when the MediaStream is muted.  If it is false, the
-                MediaRecorder should record silence (for audio) or black frames(for video)
-                when the source MediaStream is muted.  The default value for this
-                property is true.</td>
-          </tr>
-
-
-
-
-
-        </tbody>
-      </table>
-        </section>
-
+<section id="properties"> <h3>MediaRecorder  Properties</h3>
+  <p> [TO DO: Provide API surface to set and read these. When that is done, this section will be removed.]</p>
+  <table class="simple">
+  <thead>
+    <tr>
+      <th>Property Name</th>
+      <th>Values</th>
+      <th>Notes</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr id="def-property-MimeType">
+      <td>MimeType</td>
+      <td><code>DOMString</code></td>
+      <td> The MIME type that was be selected as the container for recording. This entry includes all the parameters to the base MimeType. The UA should be able to play back any of the MIME types it supports for recording. For example, it should be able to display a video recording in the HTML &lt;video&gt; tag. The default value for this property is platform-specific.</td>
```

Comment:
I see this was just moved and reformatted in this commit, but might want to fix anyway :)